test/integration/pack/test_pack.py

```diff
@@ -343,6 +343,29 @@ def test_git_version_result_filename(cartridge_cmd, project_without_dependencies
     assert expected_filename in os.listdir(tmpdir)
 
 
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_filename_flag(cartridge_cmd, project_without_dependencies, tmpdir, pack_format):
+    project = project_without_dependencies
+
+    # Project is initialized as a git repo with version tag `0.1.0`.
+    repo = git.Repo(project.path)
+    assert not repo.bare
+
+    git_version = repo.git.describe('--tags', '--long')
+    assert git_version.startswith('0.1.0-0')
+    expected_filename = "test_filename"
```

Comment:
Let's provide an example that is a bit closer to real one. At least, please add an extension (`.tar.gz`).

Reply:
fixed